source/install/trouble_mysql.rst

```diff
@@ -0,0 +1,250 @@
+.. _mysql-installation-troubleshooting:
+
+MySQL Installation Troubleshooting
+----------------------------------
+
+Before you can run the Mattermost server, you must first install and
+configure a database. You can then start Mattermost by navigating to the
+``/opt/mattermost/bin`` directory and entering the command
+``sudo -u mattermost ./platform``. If the Mattermost server cannot
+connect to the database, Mattermost will fail to start. This section deals with
+MySQL database issues that you may encounter when you start up
+Mattermost for the first time.
+
+How you install MySQL varies depending upon which Linux distribution you
+use. But once MySQL is installed the configuration instructions are the
+same for all supported distributions. You
+must create a ``mattermost`` database
+and a ``mattermost`` database user. Failure to create these database
+objects or improperly referencing them from the Mattermost configuration
+file, ``/opt/mattermost/config/config.jason``, causes Mattermost to
+fail. The troubleshooting tips given here deal with these specific
+issues.
+
+Before proceeding confirm that your MySQL server is running. You can do
+this by issuing the command: ``mysqladmin -u root -p status``. When
+prompted, enter your password. If MySQL is running you should see output
+like the following: ::
+
+    Uptime: 877134  Threads: 1  Questions: 9902  Slow queries: 0  Opens: 522  
+    Flush tables: 1  Open tables: 371  Queries per second avg: 0.011
+
+If MySQL is not running, review the instructions for installation on
+your distribution.
+
+.. warning:: Some of the commands used in this section alter the database. **Use these commands only if your Mattermost installation has failed. Do not directly manipulate the MySQL database for a working Mattermost installation**.
+
+The mattermost Database
```

Comment:
I know this is the name of the database that we use internally, but I think it looks kind of weird to have it lower case in headings since it's easy to confuse with the product name itself. I think it would be fine to have it capitalized here and refer to it in lower case elsewhere.

It might actually make sense to separate the heading from the database name in case some users have decided to give their database a different name